wordpress-mobile/WordPress-iOS
# People Management: Fixing cache glitches

#### Details:
In this PR we're updating `PeopleService` so that it uses the specified Core Data context. Several ViewControllers have been updated to use a Main-Derived-Context. We're effectively using CoreData as an ephemeral cache, and the Users / Followers won't be persisted to disk.

Reasons for doing so are:
- Users / Followers have a paging mechanism
- In order to nuke remotely-deleted-users, we're already purging the local collection of Users upon refresh.
- Even with Purging, there are several corner case scenarios that aren't being properly handled (such as accessing a user that was already deleted, before the remote response came back).

#### To test:
1. Log into a dotcom account
2. Open any blog in which you're an admin
3. Tap over People

Please, try Adding / Editing / Removing Users. Verify that everything works as expected.

Needs review: @
